lighthouse-core/audits/accessibility/select-name.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+  * @fileoverview Ensures every select element has a label.
+  * See base class in axe-audit.js for audit() implementation.
+  *
+  * See PR where select-name was split out from label:
+  * @url https://github.com/dequelabs/axe-core/pull/2448
+  */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if all select elements have corresponding label elements. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: 'Select elements have associated labels',
```

Comment:
```suggestion
  title: '`<select>` elements have associated labels',
```

so translators dont touch it